lib/assert.js

```diff
@@ -109,7 +109,7 @@ assert.ok = ok;
 // The equality assertion tests shallow, coercive equality with
 // ==.
 // assert.equal(actual, expected, message_opt);
-
+/* eslint-disable no-restricted-properties */
```

Comment:
Nit: Maybe `// eslint-disable-next-line no-restricted-properties` would be better here so that the rule isn't disabled for the rest of the file. (I'm assuming it's intended to only be disabled for this line, rather than for the rest of the file.)